src/token/signature.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <stdint.h>   // uint*_t
+#include <stddef.h>   // size_t
+#include <stdbool.h>  // bool
+#include <string.h>
```

Comment:
Nice touch on the "includes" above, but is string.h used in this header? I'd appreciate a comment with the used types, otherwise it's better if it's only included on the .c files, no?

Reply:
It's for the `memmove`, but it's used only on the signature.c, i'm moving to the correct file.

Reply:
Thanks!